dbinit/src/main/liquibase/updates/changelog-0003-yd-378.yml

```diff
@@ -0,0 +1,46 @@
+databaseChangeLog:
+- changeSet:
+    id: 1485291488594-1
+    author: Bert (generated and altered)
+    changes:
+    - addColumn:
+        columns:
+        - column:
+            name: app_last_opened_date
+            type: date
+            defaultValueDate: "2017-01-25"
```

Comment:
Maybe add a comment that a value is required, and this is used to set existing records to a date.